src/stip/common/const.py

```diff
@@ -15,6 +15,8 @@
 STIP_SNS_REFERRED_URL_KEY = 'Referred URL'
 STIP_SNS_STIX2_PACKAGE_ID_KEY = 'STIX2 Package ID'
 
+# SNS feeds.views
+HASHTAG_CHARACTER_LENGTH_LIMIT = 100
```

Comment:
I thought it would be better to use “STIP _ SNS ” like other constant names.

---

他の定数名と同じように `STIP_SNS` を付けた方いいかもしれません。

Reply:
It is not necessary to start with `STIP_SNS_`, because the constat value is used not only `stip-sns` but `stip-rs` and/or  other repositories.

I would like to recommend to set the name `MAX_HASHTAG_LENGTH`.

Reply:
I fixed it.